clients/privacy-center/cypress/e2e/consent.cy.ts

```diff
@@ -283,7 +283,8 @@ describe("Consent settings", () => {
       });
     });
 
-    it("reflects their choices using fides.js", () => {
+    // TODO (PROD-1597): re-enable this test; it's failing because overlay is not disabled!
+    it.skip("reflects their choices using fides.js", () => {
```

Comment:
Was this failing before or a new failure?

Reply:
I think it's been there for awhile, I suspect it probably fails in CI every now and then. I noticed more when I was running these tests locally with `it.only` and checking things out - I think there's a race condition here where, if the test runs fast enough, it finishes assertions before `fides.js` finishes initializing and starts calling the privacy experience API, etc.

Reply:
ah wow, this is good to know. Sounds like we need to be more explicit about waiting for a FidesInitialized event or something before proceeding with test assertions

Reply:
These tests are kinda the opposite of that - I think they were written in the era where FidesJS was purely static, so they don't worry about it loading a client-side experience and then changing to new notices 👍 

The gotcha is that the tests _think_ they set `OVERLAY_ENABLED: false` but I think there's a `cy.reload()` in there that causes the state to be lost, etc...

If I can't find a simple fix I'll carve out a follow-up tick